ui: fix word-wrapping and tooltip spacing on jobs page

celiala opened the pull request:
- Fixed spacing for tooltip copy
- Fixed styling bug that was introduced in #35651: Job description
  previously truncated long job descriptions in collapsed view.

Tooltip copy - before:

<img width="300" alt="jobs before" src="https://user-images.githubusercontent.com/3051672/54454081-0be36e00-472f-11e9-8fc2-7e59b4565d51.png">

Tooltip copy - after:

<img width="300" alt="fix tooltip-spacing" src="https://user-images.githubusercontent.com/3051672/54454082-0be36e00-472f-11e9-9683-3b63490fbfe1.png">

Job description - before:

<img width="400" alt="broken text-wrapping" src="https://user-images.githubusercontent.com/3051672/54454213-58c74480-472f-11e9-81fd-7e075510d96f.png">

Job description - after:

<img width="400" alt="fix text-overflow" src="https://user-images.githubusercontent.com/3051672/54455816-16076b80-4733-11e9-94e5-f24312ac6ff1.png">

Release note: None

Fixes: #35604

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/35797)
<!-- Reviewable:end -->

RaduBerinde commented on pkg/ui/src/views/jobs/index.tsx:
```diff
@@ -200,7 +200,7 @@ const jobsTableColumns: ColumnDescriptor<Job>[] = [
       // is a human-readable message. Otherwise job.description is a SQL
       // statement.
-      const additionalStyle = (job.statement ? "" : "--sql");
+      const additionalStyle = (job.statement ? "" : " sql");
```
I know very little CSS so sorry if it's a stupid question but shouldn't this be `.sql` ? Can you verify the font is still non-monospace for jobs with description?

celiala replied:
Doh :/ - thank you for asking this! Just pushed up a fix - RFAL, thank you!

> shouldn't this be `.sql` ?

It doesn't have to be `.sql` on the HTML side: the `.` specifies a className (`.className`), vs an id (`#idName`) -- but you were right to note that the font wasn't monospace for SQL statements! :)

> Can you verify the font is still non-monospace for jobs with description?

Double-checked that font styles are as expected for both SQL and human-readable descriptions:

Human-readable (normal font):
<img width="300" src="https://user-images.githubusercontent.com/3051672/54456101-b8275380-4733-11e9-91f0-75ada828080d.png">

SQL (monospace font):

<img width="300" src="https://user-images.githubusercontent.com/3051672/54456132-cc6b5080-4733-11e9-8869-9c0edb3cc4f7.png">

celiala committed:
ui: fix word-wrapping and tooltip spacing on jobs page

- Fixed spacing for tooltip copy
- Fixed styling bug that was introduced in #35651: Job description
  previously truncated long job descriptions in collapsed view.

Tooltip copy - before:

<img width="300" alt="jobs before" src="https://user-images.githubusercontent.com/3051672/54454081-0be36e00-472f-11e9-8fc2-7e59b4565d51.png">

Tooltip copy - after:

<img width="300" alt="fix tooltip-spacing" src="https://user-images.githubusercontent.com/3051672/54454082-0be36e00-472f-11e9-9683-3b63490fbfe1.png">

Job description - before:

<img width="400" alt="broken text-wrapping" src="https://user-images.githubusercontent.com/3051672/54454213-58c74480-472f-11e9-81fd-7e075510d96f.png">

Job description - after:

<img width="400" alt="fix text-overflow" src="https://user-images.githubusercontent.com/3051672/54455816-16076b80-4733-11e9-94e5-f24312ac6ff1.png">

Release note: None

Fixes: #35604

RaduBerinde commented:
LGTM, thanks!

vilterp reviewed (approved):
Nice fix!

celiala commented:
TFTRs!

bors r+
